app/domain/authentication/authn_jwt/jwt_claim.rb

```diff
@@ -0,0 +1,15 @@
+module Authentication
+  module AuthnJwt
+
+    # This class instance holds a JWT standard claim which is
+    # mandatory to validate.
```

Comment:
```suggestion
    # This class instance holds a JWT standard claim
```

the class represents a jwt claim. you are using it as mandatory in another place but that's something else.